.github/workflows/build.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
           version=$(yq '(.version|split("-"))[0]' rockcraft.yaml)
           base=$(yq '(.base|split(":"))[1]' rockcraft.yaml)
           tag=${version}-${base}_edge
-          sed -i "s/${app_version}/${tag}/g" rockcraft.yaml
+          sed -i "s/${app_version}/${base}/g" rockcraft.yaml
```

Comment:
Underscore no longer accepted. Do we really need to change the version on the fly?

Reply:
re: `Do we really need to change the version on the fly?`

Otherwise how to set OCI factory compatible version/tag (`8.0.33-22.04_edge`)?

Reply:
> Otherwise how to set OCI factory compatible version/tag (`8.0.33-22.04_edge`)?

The registry tag is set in the publish workflow [here](https://github.com/canonical/charmed-postgresql-rock/blob/14-22.04/.github/workflows/publish.yaml?plain=1#L46-L48). I don't think we need to change the version in `rockcraft.yaml` at all with our current release pipeline.

Reply:
I am not sure why it is here. BTW, MongoDB ROCK is not touching it: https://github.com/canonical/charmed-mongodb-rock/blob/5-22.04/.github/workflows/publish.yaml